hardhat.config.ts

```diff
@@ -55,13 +110,80 @@ const config: HardhatUserConfig = {
       },
     },
   },
-  gasReporter: {
-    enabled: process.env.REPORT_GAS !== undefined,
-    currency: "USD",
+  baseURIs: {
+    // mainnets
+    ethereum: "https://tableland.network/tables/",
+    optimism: "https://tableland.network/tables/",
+    polygon: "https://tableland.network/tables/",
+    // testnets
+    "ethereum-rinkeby": "",
+    "ethereum-goerli": "https://testnet.tableland.network/tables/",
+    "optimism-kovan": "https://testnet.tableland.network/tables/",
+    "polygon-mumbai": "https://testnet.tableland.network/tables/",
+    // devnets
+    "ethereum-rinkeby-staging": "",
+    "optimism-kovan-staging": "https://staging.tableland.network/tables/",
+    localhost: "http://localhost:8080/tables/",
   },
-  etherscan: {
-    apiKey: process.env.ETHERSCAN_API_KEY || "",
+  proxies: {
```

Comment:
manage existing proxies here

Reply:
nice! Reminder to us all: we'll have to update after we deploy